apps/bttester/src/btp_gap.c

```diff
@@ -2000,6 +2001,44 @@ set_filter_accept_list(const void *cmd, uint16_t cmd_len,
     return BTP_STATUS_SUCCESS;
 }
 
+#if MYNEWT_VAL(BLE_EXT_ADV)
+static uint8_t
+set_ext_advertising(const void *cmd, uint16_t cmd_len,
```

Comment:
This is not valid implementation of this command. From BTP spec:  "This command is used to enable/disable Extended Advertising when Start/Stop Advertising is used."

So it should be used to tune if advertising is using legacy or ext_adv PDUs, not to start advertising

Reply:
Done